src/Psl/Internal/Loader.php

```diff
@@ -121,6 +121,7 @@ final class Loader
         'Psl\Vec\chunk',
         'Psl\Vec\chunk_with_keys',
         'Psl\Vec\concat',
+        'Psl\Vec\drop',
```

Comment:
Not sure what's the correct order for these, they don't seem to be sorted alphabetically.

Reply:
it's random :p